src/python/tests/test_repair_mlss_tags.py

```diff
@@ -39,21 +40,21 @@ class TestRepairMLSSTags:
 
     """
 
-    dbc = None  # type: DBConnection
+    dbc: DBConnection | None = None
```

Comment:
```suggestion
    dbc: DBConnection = None
```

Reply:
Whilst the code will still work, `mypy` will complain if you say `dbc` type does not allow `None` values.

Reply:
@JAlvarezJarreta, `mypy` didn't complain when I tried this suggestion locally, and it is currently complaining as a result of this type annotation. @ens-sb, why not try my suggestion? If `mypy` complains, I'd be happy to make an alternative suggestion.

Reply:
@twalsh-ebi @JAlvarezJarreta Maybe differing mypy versions cause this discrepancy? In any case I will try the susggestion and backtrack if fails.